src/test/groovy/com/intuit/graphql/orchestrator/integration/GraphQLOrchestratorInterfaceSpec.groovy

```diff
@@ -0,0 +1,108 @@
+package com.intuit.graphql.orchestrator.integration
+
+import com.intuit.graphql.orchestrator.*
+import graphql.ExecutionInput
+import graphql.execution.AsyncExecutionStrategy
+import helpers.BaseIntegrationTestSpecification
+
+class GraphQLOrchestratorInterfaceSpec extends BaseIntegrationTestSpecification {
```

Comment:
Lets not prefix tests with GraphQLOrchestrator, instead the name based on usecase

Reply:
Groovy unit test cases have been renamed accordingly.